dataset/tinysnbjson/copy.cypher

```diff
@@ -0,0 +1,10 @@
+load from 'dataset/tinysnbjson/vPerson.json' create (:person {ID: ID, fName: fName, gender: gender, isStudent: isStudent, isWorker: isWorker, age: age, eyeSight: eyeSight, birthdate:cast(birthdate as date), registerTime:cast(registerTime as timestamp), lastJobDuration:cast(lastJobDuration as interval), workedHours: workedHours, usedNames: usedNames, courseScoresPerTerm: courseScoresPerTerm, grades: grades, height: height, u: cast(u as uuid)});
```

Comment:
Should we suppport such grammar as well?
COPY person FROM 'vPerson.json' (ARRAY true);

Reply:
Yes, that's the plan, but scanning to a table isn't intended to be in this PR